doc/content/verification_and_validation/ver-1kc-2.md

```diff
@@ -0,0 +1,84 @@
+
+As shown in [ver-1kc-2_mass_conservation_k10], mass is conserved between the two enclosures over time for all species. The variation in mass is only $0.4$ % for T$_2$ and H$_2$. This variation in mass can be further minimized by refining the mesh, i.e., increasing the number of segments in the domain.
+
+!media comparison_ver-1kc-2.py
+       image_name=ver-1kc-2_comparison_time_k10.png
+       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=ver-1kc-2_comparison_time_k10
+       caption=Evolution of species concentration over time governed by Sieverts' law with $K = \frac{10}{\sqrt{RT}}$.
+
+!media comparison_ver-1kc-2.py
+       image_name=ver-1kc-2_concentration_ratio_T2_k10.png
+       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=ver-1kc-2_concentration_ratio_T2_k10
+       caption=Tritium concentrations ratio between enclosures 1 and 2 at the interface for $K = \frac{10}{\sqrt{RT}}$. This verifies TMAP8's ability to apply Sieverts' law across the interface.
+
+!media comparison_ver-1kc-2.py
+       image_name=ver-1kc-2_concentration_ratio_H2_k10.png
+       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=ver-1kc-2_concentration_ratio_H2_k10
+       caption=Dihydrogen concentrations ratio between enclosures 1 and 2 at the interface for $K = \frac{10}{\sqrt{RT}}$. This verifies TMAP8's ability to apply Sieverts' law across the interface.
+
+!media comparison_ver-1kc-2.py
+       image_name=ver-1kc-2_concentration_ratio_HT_k10.png
+       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=ver-1kc-2_concentration_ratio_HT_k10
+       caption=Tritium hybride concentrations ratio between enclosures 1 and 2 at the interface for $K = \frac{10}{\sqrt{RT}}$. This verifies TMAP8's ability to apply Sieverts' law across the interface.
+
+!media comparison_ver-1kc-2.py
+       image_name=ver-1kc-2_mass_conservation_k10.png
+       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=ver-1kc-2_mass_conservation_k10
+       caption=Total mass conservation across both enclosures over time for $K = \frac{10}{\sqrt{RT}}$.
+
+## Input files
+
+!style halign=left
+The input file for this case can be found at [/ver-1kc-2.i]. To limit the computational costs of the test cases, the tests run a version of the file with a coarser mesh and less number of time steps. More information about the changes can be found in the test specification file for this case [/ver-1kc-2/tests].
```

Comment:
You currently do not use heavy tests, so this description is not representative of what is currently going on in `/ver-1kc-2/tests`. If tests take a long time to run, then you'll indeed want to reduce computational costs for basic, fast running tests, and create heavy tests. But I want to note that this is currently not the case.